common/lib/xmodule/xmodule/course_module.py

```diff
@@ -834,6 +834,16 @@ class CourseFields(object):
         scope=Scope.settings,
     )
 
+    social_sharing_url = String(
+        display_name=_("Social Media Sharing URL"),
+        help=_(
+            "A fully-qualified URL used by the social sharing icons on the dashboard (if enabled) "
```

Comment:
How about: 
If dashboard social sharing and custom course URLs are enabled, you can provide a URL (such as the URL to a course About page) that social media sites can link to. URLs must be fully qualified. For example: http://www.edx.org/course/Introduction-to-MOOCs-ITM001#.
@marcotuts @mattdrayer Can you pls especially comment on whether the example URL is an accurate/realistic example? Thanks!